rules/windows/command_and_control_remote_file_copy_powershell.toml

```
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2020/11/30"
+maturity = "production"
+updated_date = "2020/11/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies powershell.exe being used to download an executable file from a remote destination.
```

```suggestion
Identifies powershell.exe being used to download an executable file from an untrusted remote destination.
```
The logic implies a list of destinations that are trusted for this activity.